rr_common/src/follower/follower.h

```diff
@@ -0,0 +1,25 @@
+#ifndef RR_COMMON_FOLLOWER_H
+#define RR_COMMON_FOLLOWER_H
+
+#include <ros/ros.h>
+#include <pcl/point_cloud.h>
+#include <pcl/PCLPointCloud2.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <pcl/filters/passthrough.h>
+#include <string>
+#include <rr_platform/speed.h>
+#include <rr_platform/steering.h>
+#include "flann/flann.hpp"
+
+float MIN_X;
```

Comment:
I would consider a better name, something that represents obstacles in this since I was initially unsure what this was used for without reading through the code.